### ISSUE TITLE
Gridt.jks file

### ISSUE THREAD

**Issue by Drvanon** [GridtNetwork/gridt-client#177]
There is this Gridt.jks file in the root directory and I truly have no clue what it is or what it does. @AndraUdrescu you placed it there, could you tell us what it's use was? If anybody has a clue, please let me now.

**Comment by AndraUdrescu** [comment]
It was generated by android studio. It has the security key for our apk.

**Comment by Drvanon** [comment]
Let's remove it.

**Comment by AndraUdrescu** [comment]
Sure, though a new file will be created once we make another apk but I don't think we are doing any android builds right now

**Comment by Drvanon** [comment]
I think that that file actually should end up in the android folder, as that is where all the android files are. The entire android apk is created on the level of that folder and when using an android program, it should not be on the level of the ionic project.

**Comment by AndraUdrescu** [comment]
Yeah, I don't know why it was created there. We can delete it since we will have to regenerate most files due to updates and changes

**Comment by Drvanon** [comment]
Can you create a quick PR?

**Comment by AndraUdrescu** [comment]
With the files deleted?

**Comment by Drvanon** [comment]
Yeah, if you know of other files that are related to this, feel free to remove them as well.
